src/pages/Home.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import { Outlet } from "react-router-dom";
-import TopNav from "../components/TopNav";
+import ThemeHandler from "../components/ThemeHandler";
 const Home = () => {
     return (
         <div>
-            <TopNav />
```

Comment:
The location of the TopNav component and the rationale for it has been discussed in a previous comment.